testdata/project-v2-addon/go.mod

```diff
@@ -9,5 +9,5 @@ require (
 	k8s.io/apimachinery v0.18.2
 	k8s.io/client-go v0.18.2
 	sigs.k8s.io/controller-runtime v0.6.0
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200512162422-ce639cbf6d4c
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200522144838-848d48e5b073
```

Comment:
Is this change really needed in this PR?

Reply:
solved here: https://github.com/kubernetes-sigs/kubebuilder/pull/1532 - It is related to the scenario that we need to  solve over it.